client/reader/following-manage/url-helper.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
I think we should squint at this file a bit and try to merge it with `/lib/url/index.js`